packages/beacon-node/src/network/core/networkCore.ts

```diff
@@ -2,15 +2,15 @@ import {PeerId} from "@libp2p/interface-peer-id";
 import {multiaddr} from "@multiformats/multiaddr";
 import {Connection} from "@libp2p/interface-connection";
 import {PublishOpts} from "@chainsafe/libp2p-gossipsub/types";
-import {routes} from "@lodestar/api";
 import {PeerScoreStatsDump} from "@chainsafe/libp2p-gossipsub/dist/src/score/peer-score.js";
+import {fromHexString} from "@chainsafe/ssz";
+import {ENR} from "@chainsafe/discv5";
+import {routes} from "@lodestar/api";
 import {BeaconConfig} from "@lodestar/config";
-import {LoggerNode} from "@lodestar/logger/node";
+import type {LoggerNode} from "@lodestar/logger/node";
```

Comment:
@lodestar/logger is a dependency of beacon-node, what is the reason for adding explicit type only import?

Reply:
On 2nd thought, maybe it makes sense. The only reason the beacon-node packages requires @lodestar/logger as dependency is because the workers need to initialize their own instance using `getNodeLogger`. I don't think this can be solved though as it will not be possible to inject the logger to the worker process.

Reply:
For type only import, it was changed before I realized that worker are requiring `getNodeLogger`.